{{cookiecutter.project_name}}/docs/template_usage.md

```diff
@@ -226,7 +211,7 @@ The `RUF100` check will remove `noqa` IDs that are no longer necessary.
 If you want to add an ID that comes from a tool other than Ruff,
 add it to Ruff’s [`external = [...]`](https://beta.ruff.rs/docs/settings/#external) setting to prevent `RUF100` from removing it.
 
-You can also disable certain error messages for the entire project.
+Alternatively, you can disable certain error messages for the entire project.
```

Comment:
I removed the “alternatively” beacuse Iafter the `RUF100` paragraph, I thought it’s no longer clear that it refers to the paragraph that’s now two ago.

Reply:
I've rewritten this section.